dfhack.init-example

```diff
@@ -177,6 +181,7 @@ tweak import-priority-category
 # Misc. UI tweaks
 tweak block-labors              # Prevents labors that can't be used from being toggled
 tweak civ-view-agreement
+tweak eggs-fertile
```

Comment:
I have mixed feelings about enabling this by default. I said that I considered it cheating when it came up for discussion when I added it, and most of the other tweaks enabled by default are bugfixes or improvements.